patches/server/0210-PlayerReadyArrowEvent.patch

```diff
@@ -28,12 +28,22 @@ index e088e5174a4666130cd1f66c751e7ab83d9f7a65..8aa5f8bacb868a3b04af6e696f9eed89
      @Override
      public ItemStack getProjectile(ItemStack stack) {
          if (!(stack.getItem() instanceof ProjectileWeaponItem)) {
-@@ -2242,7 +2253,7 @@ public abstract class Player extends LivingEntity {
+             return ItemStack.EMPTY;
+         } else {
+-            Predicate<ItemStack> predicate = ((ProjectileWeaponItem) stack.getItem()).getSupportedHeldProjectiles();
++            Predicate<ItemStack> predicate = (item) -> ((ProjectileWeaponItem) stack.getItem()).getSupportedHeldProjectiles().test(item) && tryReadyArrow(stack, item); // Paper
```

Comment:
You should combine these predicates with the instance `and(Predicate)` method instead.

Reply:
Updated using this method.